eng/pipelines/prepare-pipelines.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
```

Comment:
With no trigger, what triggers it? Manual?

Reply:
The pipeline is triggered by a service team which is onboarding a new service for the first time. The way this will work is that Check Enforcer will detect that a PR hasn't run any pipelines. It will then add a comment to the PR with some helpful tips, one of those tips is to "prepare pipelines" for the new service, which the customer would do by issuing this command:

```/azp run prepare-pipelines```

This file is the definition that sits behind this.